src/Module/Settings/Account.php

```diff
@@ -286,6 +272,25 @@ protected function post(array $request = [])
 			}
 			DI::pConfig()->set(local_user(), 'system', 'notify_type', $notify_type);
 
+			if (!($notify_type & 72)) {
```

Comment:
Please create constants for these bitmask values, I have no idea what this is doing so will you 6 months from now when someone complains about the notifications again.